Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -72,6 +72,18 @@ configuration:
   name: fetch_time
   required: false
   type: 0
+- defaultvalue: earliest_time
+  display: Earliest time to fetch (The name of the Splunk field whose value defines
+    the query's earliest time to fetch.)
+  name: earliest_fetch_time_fieldname
+  required: false
+  type: 0
+- defaultvalue: latest_time
+  display: Latest time to fetch (The name of the Splunk field whose value defines
+    the query's latest time to fetch.)
+  name: latest_fetch_time_fieldname
+  required: false
+  type: 0
```

Comment:
They were on the pre-release so I thought maybe it was an improvement or a customer ask for.. Should I remove it? The default are same as it was hard coded on SplunkPy, so customer does not need to edit if he just update the pack, it will not change nothing for him.